documentation/Learn-Beton/Frequently-Asked-Questions.md

```diff
@@ -161,6 +161,20 @@ These events will set a specific tag. Now when the player starts the conversatio
 should check (using the `globaltag` condition) which tag is currently set and give the player different quests based on that.
 Of course, the static folder event also needs to remove the the current tag before setting a new one.
 
+## Global Quests (all players work together)
+
+There is no super easy way to do this yet. One would do this:
+Create an objective that is immediately fired upon the first action (this means setting the amount to one for most objectives).
+That objective must be `persistent` so it restarts immediately upon completion.
+It would trigger a `globalpoint` event that increases some globalpoint variable by one.
+That globalpoint variable tracks the players combined progress as it is the same for all of them.
+Therefore, there must also be some sort of `globalpoint`condition that checks whether the "global objective" has been completed.
+It must be checked in a "checkCompletion" event that is run alongside the globalpoint event. It could look like this:
```

Comment:
This part is very `it` heavy (5 times used, three of those at start of sentence).